applications/matter_weather_station/src/weather-station.matter

```diff
@@ -157,7 +157,12 @@ server cluster Basic = 40 {
   readonly attribute int32u softwareVersion = 9;
   readonly attribute char_string<64> softwareVersionString = 10;
   readonly attribute char_string<16> manufacturingDate = 11;
+  readonly attribute char_string<32> partNumber = 12;
```

Comment:
Are those changes needed? Last time I configured these attributes to match what we test in SVE.

Reply:
Ok, I automatically aligned it to other samples, but we can also keep it consistent with SVE. Did you mean only basic cluster or all? I reverted changes for Basic cluster.

Reply:
I mean all. We don't implement diagnostic logs, Thread metrics etc. If they are enabled in other samples, we should rather disable them there.

Reply:
I think we should work on creating some kind of array of supported features for our samples then. Reviewing all NCS vs upstream differences in clusters and deciding should we support it or not on every upmerge is not that easy. I can try to revert some configuration, but honestly I'm not sure which part of it is useless and which might be used somewhere under the hood.